### PR TITLE
style: shellcheck all .sh files

### PR DESCRIPTION
**Pull request by actionless**

### CODE REVIEW

File: render-assets.sh [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 fi
 
-if [[ ! "$(which optipng 2> /dev/null)" ]]; then
+if [[ ! "$(command -v optipng)" ]]; then
   echo "'optipng' needs to be installed to optimize the resulting PNG!"
 fi
 
 chrome() (
-  cd src/chrome
+  cd ./src/chrome
```
Comment (nana-4) [inline render-assets.sh]: Why is `./` necessary only for this? shellcheck does not seem to give warning even without it.

Comment (actionless) [reply render-assets.sh]: i was thinking it could be more readable but it's indeed not important

Comment (nana-4) [reply render-assets.sh]: I see. I just wondered why this is only in `src/chrome` but not in `src/gtk` and `src/gtk-2.0` :)

**Comment by nana-4** [comment]
LGTM. Thanks a lot!

**Comment by actionless** [comment]
thanks for reviewing!

**Pull request merged**